🤖 Auto-merge documentation from branches

buger opened the pull request:
### **User description**
## 📚 Documentation Merge Summary

This PR contains automatically merged documentation from multiple branches.

**Triggered by:**
- **Branch:** N/A
- **Commit:** N/A
- **PR:** N/A (direct push)

**Deployment Details:**
- **Generated from:** `branches-config.json`  
- **Run ID:** 19919159628
- **Subfolder:** `(root deployment)`
- **Timestamp:** $(date)

### Changes Include:
- ✅ Merged documentation from multiple branches
- ✅ Generated unified `docs.json` configuration
- ✅ Updated assets and content structure
- ✅ Cleaned up outdated version folders

---

🚦 **This PR will be processed by merge queue to ensure proper validation and ordering.**

Previous deployment PRs have been automatically closed to prevent conflicts.


___

### **PR Type**
Documentation


___

### **Description**
- Add guide for importing custom credentials

- Update docs navigation to include new guide

- Provide API-based bulk import examples

- Add nightly and stable portal variants


___

### Diagram Walkthrough


```mermaid
flowchart LR
  docsjson["docs.json navigation"] -- "add links" --> portalGuide["tyk-developer-portal/import-custom-credentials.mdx"]
  docsjson -- "add nightly links" --> nightlyGuide["nightly/tyk-developer-portal/import-custom-credentials.mdx"]
  portalGuide -- "UI + API steps" --> readers["Portal admins & users"]
  nightlyGuide -- "nightly variant" --> readers
```



<details> <summary><h3> File Walkthrough</h3></summary>

<table><thead><tr><th></th><th align="left">Relevant files</th></tr></thead><tbody><tr><td><strong>Documentation</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>docs.json</strong><dd><code>Add import custom credentials to navigation (stable/nightly)</code></dd></summary>
<hr>

docs.json

<ul><li>Add <code>tyk-developer-portal/import-custom-credentials</code> to Guides.<br> <li> Add nightly path <br><code>nightly/tyk-developer-portal/import-custom-credentials</code>.<br> <li> Update both stable and nightly "Consuming APIs > Guides" sections.</ul>


</details>


  </td>
  <td><a href="https://github.com/TykTechnologies/tyk-docs/pull/1109/files#diff-b7b4169fa5b2e02aa17d40553a73c5d9520086753c501eeb926c63b162bb6d9e">+4/-2</a>&nbsp; &nbsp; &nbsp; </td>

</tr>

<tr>
  <td>
    <details>
      <summary><strong>import-custom-credentials.mdx</strong><dd><code>New nightly guide: import custom credentials</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

nightly/tyk-developer-portal/import-custom-credentials.mdx

<ul><li>New guide on associating products with custom credentials.<br> <li> Includes UI walkthrough, verification, and API examples.<br> <li> Notes availability since portal v1.12.0.<br> <li> Embeds demo video and illustrative images.</ul>


</details>


  </td>
  <td><a href="https://github.com/TykTechnologies/tyk-docs/pull/1109/files#diff-7fab822dc85e245eed1764af91c359eae7aeea63f28f3cdefd6ace988a2d1c30">+145/-0</a>&nbsp; </td>

</tr>

<tr>
  <td>
    <details>
      <summary><strong>import-custom-credentials.mdx</strong><dd><code>New guide: import custom credentials</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

tyk-developer-portal/import-custom-credentials.mdx

<ul><li>New stable guide mirroring nightly content.<br> <li> Step-by-step UI instructions and verification steps.<br> <li> API endpoints for bulk imports documented.<br> <li> Availability note and embedded demo video.</ul>


</details>


  </td>
  <td><a href="https://github.com/TykTechnologies/tyk-docs/pull/1109/files#diff-d14e03e0339e63470702b1693872334cc3b5072581f3fc7fe76f6929a4d45794">+145/-0</a>&nbsp; </td>

</tr>
</table></td></tr></tr></tbody></table>

</details>

___

marksou commented on tyk-docs/content/getting-started/installation/with-tyk-multi-cloud/create-an-account.md:
@sedkis I think this PR is no longer required, as Tyk Cloud signup has superseeded the above?

buger committed:
🤖 Auto-merge documentation from branches

Generated from branches-config.json at

- Merged documentation from multiple branches
- Generated unified docs.json
- Updated assets and content structure
